journald/journald.cpp

```diff
@@ -29,7 +32,8 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
 {
 public:
   JournaldLoggerProcess(const Flags& _flags)
-    : flags(_flags)
+    : ProcessBase(process::ID::generate("journald-logger")),
```

Comment:
Alright. Thanks for the explanation. We can merge it with the duplication, but you're going to have to keep an eye on them to make sure they stay in sync.